scv-prototype-frontend/src/app/views/Login/Login.jsx

```diff
@@ -137,4 +135,4 @@ const Login = ({ location }) => {
   );
 };
 
-export default Login;
+export default withRouter(Login);
```

Comment:
We can use hooks for router stuff instead of higher order components. ex: `useLocation` and `useHistory`